AGILE/Interpreter.cs

```diff
@@ -176,8 +172,18 @@ public void Tick()
                 byte previousScore = state.Vars[Defines.SCORE];
                 bool soundStatus = state.Flags[Defines.SOUNDON];
 
-                // Continue scanning LOGIC 0 while the return value is above 0, indicating a room change.
-                while (NewRoom(commands.ExecuteLogic(0))) ;
+                // Continue scanning LOGIC 0 while NewRoom() return value is true, indicating a room change.
+                bool executeLogicLoop = true;
+                while (executeLogicLoop)
```

Comment:
I'm iffy with this change.. not sure whats the correct way to do this

but the `previousScore = state.Vars[Defines.SCORE];` is important to fixing the PQ bug

without `previousScore = state.Vars[Defines.SCORE];`, `previousScore` will be 0 and thus trigger an `UpdateStatusLine`